ProjectDataCore.Data/Services/Alert/AlertService.cs

```diff
@@ -0,0 +1,60 @@
+﻿using ProjectDataCore.Data.Structures.Model.Alert;
+using System.Timers;
+
+namespace ProjectDataCore.Data.Services.Alert
+{
+    public class AlertService : IAlertService
+    {
+        public List<AlertModel> Alerts { get; set; } = new();
+        public event EventHandler AlertsChanged;
+
+        public ActionResult CreateAlert(string message, AlertType alertType, bool enableTimer, int duration)
+        {
+            AlertModel alertModel = new(alertType, message, this, enableTimer, duration);
+            Alerts.Add(alertModel);
+            return TriggerEvent();
+        }
+
+        public ActionResult CreateErrorAlert(string message, bool enableTimer = false, int duration = 5000)
+        {
+            return CreateAlert(message, AlertType.Error, enableTimer, duration);
+        }
+
+        public ActionResult CreateInfoAlert(string message, bool enableTimer = false, int duration = 3200)
+        {
+            return CreateAlert(message, AlertType.Info, enableTimer, duration);
+        }
+
+        public ActionResult CreateSuccessAlert(string message, bool enableTimer = false, int duration = 3200)
+        {
+            return CreateAlert(message, AlertType.Success, enableTimer, duration);
+        }
+
+        public ActionResult CreateWarnAlert(string message, bool enableTimer = false, int duration = 3200)
+        {
+            return CreateAlert(message, AlertType.Warn, enableTimer, duration);
+        }
```

Comment:
Alerts, specifically the error alerts, should be able to take a list of strings to match the error field in the ActionResult.